doc/en/how-to/cache.rst

```diff
@@ -277,20 +277,158 @@ You can always peek at the content of the cache using the
 
     $ pytest --cache-show
     =========================== test session starts ============================
-    platform linux -- Python 3.x.y, pytest-6.x.y, py-1.x.y, pluggy-0.x.y
+    platform linux -- Python 3.x.y, pytest-7.x.y, py-1.x.y, pluggy-0.x.y
     cachedir: $PYTHON_PREFIX/.pytest_cache
     rootdir: $REGENDOC_TMPDIR
     cachedir: $PYTHON_PREFIX/.pytest_cache
     --------------------------- cache values for '*' ---------------------------
     cache/lastfailed contains:
-      {'test_50.py::test_num[17]': True,
+      {'a/test_db.py::test_a1': True,
```

Comment:
Hmmm with the documentation moved we are getting cache data from other runs. @RonnyPfannschmidt can we use "regendoc wipe" to fix this?

Reply:
indeed, i believe this one needs a wipe

Reply:
What does that mean exactly?

Reply:
Hmm looking at it more carefully, we already have a `regendoc:wipe` in `The new config.cache object` a few lines above. Seems like `regendoc:wipe` might not be deleting the `.pytest_cache` directory?